src/components/proofs/ConnectAccount.tsx

```diff
@@ -40,14 +43,36 @@ const bottomWrapper = classnames(
 
 export default function () {
   const { walletLoading, needNetworkChange } = useSnapshot(WalletStore)
+  const zkProofText =
+    'In your wallet(s), you have NFTs that can point back to your identity (aka, getting doxxed). But with ZK proof, you can verify ownership of NFTs while staying pseudonymous.'
 
   return (
     <div className={walletContainer}>
       <div className={contentWrapper}>
         <HeaderText accent>First</HeaderText>
         <div className={bottomWrapper}>
           <BodyText center>
-            Connect a wallet with NFTs to create a ZK proof.
+            <span>
```

Comment:
Do we need this `span` wrapper?

Reply:
oh, you're right, we could get rid of this aswell

Reply:
ty